### PR TITLE
Minor consistency tweaks and optimizations

### LINKED ISSUE #4: Banner image is too large
It's 5.7MB. Somthing <<1MB would make loading much faster.

### LINKED ISSUE #5: Consistent page/content titles
Currently the `contact` and `contest` pages have both a page title (in white on the banner image) and a content title. One should be enough.

The CFP page is missing the page title.

### LINKED ISSUE #6: Add date to `contest` page
Currently it's only in the news section, but it could/should be more pronounced. Anyway that's not the first place I would look to find this info.

### LINKED ISSUE #7: `news` pages are missing the banner image
Maybe it's intentional? But the change in style is confusing

### PR DESCRIPTION
**Pull request by verwoerd**
Improvements based on the issues. Fixes #4, Fixes #5, Fixes #6 and Fixes #7.

### CODE REVIEW

**Comment by verwoerd** [comment]
also moved the git lfs for images. Be sure to run `git lfs install` once(https://git-lfs.github.com/)

**Review by maartenweyns — APPROVED** [review]

**Comment by mpsijm** [comment]
> Be sure to run `git lfs install` once

@verwoerd Can you add this to the `README`?

File: README.md [hunk 1]
```diff
@@ -1,6 +1,10 @@
 # NWERC 2022 Website
 This is the website for NWERC 2022, created with [Hugo](https://gohugo.io/).
 
+## Before you Start
+This repository use [lfs](https://git-lfs.github.com/) for large files. make sure you have lfs installed and configured
+on your local system.
+
 ## Development
 - Clone this repository including submodules: `git clone --recurse-submodules <url>`
```
Comment (mpsijm) [inline README.md:9]:
```suggestion
- Clone this repository including submodules: `git clone --recurse-submodules <url>`
- Run `git lfs install` to initialize LFS hooks for this repository
```

File: README.md [hunk 2]
```diff
@@ -1,7 +1,12 @@
 # NWERC 2022 Website
 This is the website for NWERC 2022, created with [Hugo](https://gohugo.io/).
 
+## Before you Start
+This repository use [lfs](https://git-lfs.github.com/) for large files. make sure you have lfs installed and configured
+on your local system.
```
Comment (mpsijm) [inline README.md:6]: Sorry, one more: missing a capital letter :innocent: 

```suggestion
This repository use [lfs](https://git-lfs.github.com/) for large files.
Make sure you have lfs installed and configured on your local system.
```

**Review by mpsijm — APPROVED** [review]

**Pull request merged**